internal/fpath/file.go

```diff
@@ -112,3 +121,53 @@ func DeepCopy(src string, target string) error {
 
 	return nil
 }
+
+// FindFiles returns file lists find by shell pattern
```

Comment:
Doc isn't accurate though 😅 It's not just a shell pattern (glob pattern, actually), but also regex.
```suggestion
// FindFiles returns a list of files as identified by the sources. Source pattern interpretation (e.g. regex or glob) is controlled by matchBy.
```